### DISCUSSION TITLE
Always hard to localize error causing line

### DISCUSSION THREAD

**Discussion by buhtz** [sphinx-doc/sphinx#12851] [General]
I get errors like this:

```
reading sources... [100%] tools

/home/user/goodCloud/my.work/bit/backintime/common/snapshots.py:docstring of snapshots.Snapshots:4: WARNING: Inline literal start-string without end-string.
looking for now-outdated files... none found
```

But line 4 in that file is not the cause. I don't get it how Sphinx does the line numbering.
This is the original file: https://github.com/bit-team/backintime/blob/dev/common/snapshots.py

Does anybody has an idea? I also would like to understand the Sphinx logic to prevent questions like this in the future and find the error line myself.

**Comment by AA-Turner** [comment]
The line points to the start of the paragraph containing the error -- this is a known issue.

A

**Reply by buhtz** [reply]
Can you link the issue please if it is "known".

> The line points to the start of the paragraph containing the error

I don't understand. What is "4" in this case?

**Reply by buhtz** [reply]
Please re-open the thread. It is not solved for me.
It is still unclear for me which one of the doc strings in that file is the cause of the problem.
